src/deadline/cinema4d_adaptor/Cinema4DClient/cinema4d_handler.py

```diff
@@ -175,6 +187,10 @@ def set_scene_file(self, data: dict) -> None:
             FileNotFoundError: If path to the scene file does not yield a file
         """
         scene_file = data.get("scene_file", "")
+
+        if locale.getpreferredencoding().lower() not in ("utf8", "utf-8"):
```

Comment:
Could we always encode and decode and avoid the branch here?

Reply:
I was thinking of not modifying filenames for UTF-8 filenames as it wouldn't change anything in most cases. Any reason to do it besides avoiding the branch?